### PR TITLE
chore: supporting infra-nginx (MAPCO-3880)

### PR DESCRIPTION
**Pull request by almog8k**

### CODE REVIEW

File: helm/values.yaml [hunk 1]
```diff
@@ -96,26 +61,72 @@ resources:
       memory: 128Mi
     requests:
       cpu: 100m
-      memory: 128Mi
+      memory: 128M
```
Comment (asafMasa) [inline helm/values.yaml:64]: check

File: helm/values.yaml [hunk 2]
```diff
@@ -96,26 +61,72 @@ resources:
...
-  timeout:
+nginx:
+  enabled: true
+  fullnameOverride: ""
+  authorization:
+    enabled: false
+  replicaCount: 1
+  image:
+    repository: nginx-otel-unprivileged
+    tag: "v1.0.0"
```
Comment (asafMasa) [inline helm/values.yaml:74]: check

File: helm/values.yaml [hunk 3]
```diff
@@ -39,55 +38,21 @@ authentication:
...
 env:
   port: 80
   targetPort: 8080
   externalPort: 8080
   protocol: TCP
   tracing:
     url: http://localhost:55681/v1/trace
-    
+
 resources:
```
Comment (asafMasa) [inline helm/values.yaml:56]: check if duplicated

File: helm/values.yaml [hunk 4]
```diff
@@ -96,26 +61,72 @@ resources:
...
-      useExternal: ''
-  cors:
+    tls:
+      enabled: false
+      useCerts: false
+      certificate: ""
+      key: ""
+      caCertificate: ""
+  ingress:
     enabled: true
```
Comment (asafMasa) [inline helm/values.yaml:123]: false

File: helm/output [hunk 5]
```diff
@@ -0,0 +1,274 @@
+---
```
[the remark this answers is not in the document]
Comment (almog8k) [reply helm/output:1]: done

File: helm/templates/_resources.tpl [hunk 6]
```diff
@@ -1,34 +1,6 @@
 {{/*
-Create service name as used by the service name label.
+Create mapproxy nginx configmap name as used by the service name label.
```
[the remark this answers is not in the document]
Comment (almog8k) [reply helm/templates/_resources.tpl:2]: done

File: helm/values.yaml (hunk 1 above)
Comment (almog8k) [reply helm/values.yaml:64]: done

File: helm/values.yaml (hunk 3 above)
Comment (almog8k) [reply helm/values.yaml:56]: deleted

**Pull request merged**